spec/requests/orders_spec.rb

```diff
@@ -7,6 +7,7 @@
 
   let!(:order) { create(:order) }
   let!(:order2) { create(:order) }
```

Comment:
@syncrou can we add a test in the spec/requests/orders_rbac_spec.rb, that will test the RBAC